lib/server.js

```diff
@@ -102,6 +102,13 @@ Server.prototype.handleRequest = function handleRequest (req, res) {
 
   // add utilites to req and res
   setupReqRes(server.options, req, res, function(err, next) {
+    if(server.options.hasOwnProperty("responseHeaders")) {
+	for (var header in server.options.responseHeaders) {
+    	     if (server.options.responseHeaders.hasOwnProperty(header)) {
+        	res.setHeader(header, server.options.responseHeaders.header);
```

Comment:
Shouldn't this be `res.setHeader(header, server.options.responseHeaders[header]);` ?